lib/jsdom/living/nodes/HTMLSelectElement-impl.js

```diff
@@ -90,6 +90,12 @@ class HTMLSelectElementImpl extends HTMLElementImpl {
     return this.options.length;
   }
 
+  get selectedOptions() {
+    return createHTMLCollection(this, () => domSymbolTree.treeToArray(this, { filter(node) {
```

Comment:
Can do, but it's not so bad. Basically just `return this._selectedOptions || (this._selectedOptions = ...);`, plus a test that el.selectedOptions === el.selectedOptions (and maybe some tests that changing selection also preseves identity).